### DISCUSSION TITLE
2024/08/toads/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#107] [General]
# 2024/08/toads/

https://xiangyun.rbind.io/2024/08/toads/

**Comment by earfanfan** [comment]
>我的父母是不读书的，农民身上的毛病，他们全都有，攀比炫耀、目光短浅、出口伤人等等。

人的品格和读不读书、是不是农民无关，你说的这些缺陷是全社会全人类都有的。太过高看知识分子身份、低看农民身份的话，有可能是一种阶级划分的意识。我们多年生活在城市里，自然习惯了追求效率，在沟通上追求效率，在做事上追求效率，往往被残酷的社会分工裹挟而不自知。可是长久生活在农村的人，做人做事所追求的未必就是效率第一，而有可能是其他。

另外，我们所生活的社会有长达几千年的被礼教约束的意识形态，不同收入阶层以及生活在不同地域的人，其名与实分离的程度是不一样的。我猜想，越是追求分工效率的地方，可能名与实分离的程度也越高，反之亦然。

**Reply by XiangyunHuang** [reply]
> 人的品格和读不读书、是不是农民无关，你说的这些缺陷是全社会全人类都有的。太过高看知识分子身份、低看农民身份的话，有可能是一种阶级划分的意识。

如果分别在农民和知识分子中取一个相当大的样本来比较的话，我相信读书会在塑造人的品行方面起作用的，至于，具体到个人，品行究竟如何还受其它三大因素影响（自我教育、社会教育和家庭教育）。知识分子身上的毛病，我也知道一些，当年整风、反右等针对知识分子的改造运动讲比较多。

> 我们多年生活在城市里，自然习惯了追求效率，在沟通上追求效率，在做事上追求效率，往往被残酷的社会分工裹挟而不自知。可是长久生活在农村的人，做人做事所追求的未必就是效率第一，而有可能是其他。

明面上我是在追求效率，实际上我其实在争取平等。平等是需要争取的、抗争的，具体的事情就是我如文中所述的那样，从吃饭（经济独立，早上的早餐我就要自己做）、穿衣（不管喜不喜欢，妈给我买的衣服就要退掉）、盖房（这就不赘述了）、婚姻（相亲之类的）等具体的事情上去做。至于父母，他们的控制远多于关爱，他们更多追求攀比炫耀，如果不这样，他们几十年的努力就得不到回报一样。以上，我是就我这个个体在谈。

> 另外，我们所生活的社会有长达几千年的被礼教约束的意识形态，不同收入阶层以及生活在不同地域的人，其名与实分离的程度是不一样的。我猜想，越是追求分工效率的地方，可能名与实分离的程度也越高，反之亦然。

文中，我说知识经验等会帮助积累财富，当然知识经验会反映到效率上，意味着收入高的阶层，对名与实的观察更加深，对社会和人性的了解也可能更深。但并不意味着这些收入高的人自己名与实很分离，更多的是他们自己心里其实很清楚，实际做的时候被迫或主动地做言行不一、知行不一的事，这一点我深有体会，不展开说了。

**Reply by earfanfan** [reply]
好奇你早上自己做早餐会吃撒？

**Reply by XiangyunHuang** [reply]
在老家一般吃面，拌点西红柿鸡蛋或者隔夜剩下的菜

**Comment by XiangyunHuang** [comment]
@hongtaoh 对了，别忘了来这里表达你的视角哦。

**Reply by hongtaoh** [reply]
这篇博客看完我有很多想说的点，全都说完的话估计又是一篇博客了。

我觉得湘云你很勇敢，把那些我看了都觉得心里难受的伤疤写出来给别人看，我觉得这本身就是一种很大的改变决心。我最近的体会是，人世间的很多道理并没有绝对的对和错，关键是适不适合自己。人只有对自己了解足够深了，才能比较熟练地运用这些道理。比如，我现在就不知道，对我来说，过去的事情到底对我有没有影响。你这篇博客提到的书里说有影响，但是《被讨厌的勇气》里就讲说过去对人没有影响，到底谁对呢？我不知道。

人都想认识世界。作为一个个体，如果对世界的认识不正确，会导致自己的判断失误，最终给自己带来损失。认识世界的话，从古至今有的方法是读书、聊天、内省。到了现代社会，多了一个互联网。问题就如你所说的，互联网上的现在比较流行的全都是让人有贪欲的，或者说只是让人及时行乐的。

**Reply by hongtaoh** [reply]
你这篇博客主题太分散了，所以我也只能抓住我比较能说得上话的来讲。你讲到的与父母在盖房子上的摩擦我其实并不是很能感同身受，因为我没有经历过类似的事情。

**Reply by XiangyunHuang** [reply]
是有些发散，我的笔力不够，紧凑不起来。我想讲的是三个方面，文章中应该说都包含的，具体如下：

1. 源起，怎么开始接触心理学的，为什么要接触心理学。

1. 心理学的第一本书，书讲什么呀，这里有选择性地讲对我影响大的点。

1. 接着，就要结合自己来谈，从具体事例到思考。

**Reply by XiangyunHuang** [reply]
> 这篇博客看完我有很多想说的点，全都说完的话估计又是一篇博客了。

哈哈，你可以挑一些你自己有感触的去写一些。

> 我觉得湘云你很勇敢，把那些我看了都觉得心里难受的伤疤写出来给别人看，我觉得这本身就是一种很大的改变决心。

谢谢你的鼓励。我遇到很挫折的事，可能当时绕过去或转移注意力，其实，将来还是会正面硬刚，一定要越过去。写这篇文章就是在正面硬刚自己的心理了。

> 我最近的体会是，人世间的很多道理并没有绝对的对和错，关键是适不适合自己。人只有对自己了解足够深了，才能比较熟练地运用这些道理。

是的，我认同。所以，读书不在多，而在质量，找到自己的路。对自己比较有影响的书，很认真地写读后感，记录下自己的心理变化过程。


> 比如，我现在就不知道，对我来说，过去的事情到底对我有没有影响。你这篇博客提到的书里说有影响，但是《被讨厌的勇气》里就讲说过去对人没有影响，到底谁对呢？我不知道。

我没看过你说的这本书。我没读过其它心理学的书（看了些别人的读后感，随手翻了点书店的书），更别说流派了。但是，我注意到有些心理学的书里面的道理是一样的，只是换个人、换个经历、换种说法而已。重要的是对建立自己的路有没有用，其它不重要。

> 人都想认识世界。作为一个个体，如果对世界的认识不正确，会导致自己的判断失误，最终给自己带来损失。认识世界的话，从古至今有的方法是读书、聊天、内省。到了现代社会，多了一个互联网。问题就如你所说的，互联网上的现在比较流行的全都是让人有贪欲的，或者说只是让人及时行乐的。

我有看到你写过两篇关于互联网产品内容的分析---[我对小红书推荐内容的思考和反思](https://hongtaoh.com/cn/2024/08/09/xhs/)和[为什么我会被这些视频吸引：一场自我剖析](https://hongtaoh.com/cn/2024/08/08/bilibili/)，我觉得特别好，知道它、认识它，自己在拒绝它的诱惑方面的能力会更强的。

**Reply by XiangyunHuang** [reply]
我今天重读，我要接受你的意见，主题太多显得太分散了。我要为自己辩驳的是，当时写这篇文章的时候想到了很多以前的事情，正因为事情太多，情绪太多，想一吐为快，章法就散乱了，再加之笔力有限。

**Reply by hongtaoh** [reply]
没事啊。这是你的文章，你有权利决定怎么写，读不读是读者的事。我当时说「太分散」，只说一个客观的描述，没有带着「文章不应该分散」这种预期。

**Reply by XiangyunHuang** [reply]
这年头对于像你这样的熟人给的意见和建议，尽管初看有点拒绝，过后看到真诚。

**Comment by zhanglj37** [comment]
很喜欢这本，后来看了一本类似的《也许你该找个人聊聊》更加强烈推荐

**Reply by XiangyunHuang** [reply]
谢谢推荐，我会找来看看的。

**Reply by XiangyunHuang** [reply]
当初这本书给我带来这么大的情绪波动，主要是两个点，其一是第一次读心理学，其二是第一次比较系统地回顾自己不堪的过往。

当我放下自己对自己的内耗后，学会自己与自己相处，我好了很多。至于与父母的关系，这个要保持适当距离，这个适当在于与他们建立原则，让他们明白一旦越过原则，那该死的控制欲，我不会忍让。为此，该吵吵，剩下的自己快速平复自己的情绪吧。哪怕他们要死要活也不能退让，为此，我已经在小事上声张自己的权利做了很多了，到今天为止，还不做原则要求就晚了。这个过程是一步步加码过来的，太不容易。好了，刚和家里大吵一次，记于此。
